services/workflows-service/src/alert/alert.service.ts

```diff
@@ -20,6 +22,32 @@ export class AlertService {
     return await this.prisma.alertDefinition.create({ data: dto as any });
   }
 
+  async updateAlertsAssignee(
+    alertIds: string[],
+    projectId: string,
+    assigneeDto: AlertAssigneeUniqueDto,
+  ): Promise<Alert[]> {
+    try {
+      return await this.alertRepository.updateMany(alertIds, projectId, {
+        data: {
+          assigneeId: assigneeDto.assigneeId,
+        },
+      });
+    } catch (error: unknown) {
```

Comment:
`unknown` is supposed to be the default.
```suggestion
    } catch (error) {
```